src/Common/Commands.Common/PowerBICmdlet.cs

```diff
@@ -53,9 +53,17 @@ private static Assembly RedirectAssemblyLoad(object sender, ResolveEventArgs arg
             string assemblyFilePath = Path.Combine(executingDirectory, requestedAssembly.Name + ".dll");
             if (File.Exists(assemblyFilePath))
             {
-                return Assembly.LoadFrom(assemblyFilePath);
+                try
+                {
+                    return Assembly.LoadFrom(assemblyFilePath);
+                }
+                catch (Exception)
```

Comment:
Do you want to return a message or something in this case?

Reply:
Because this is in a static method, we can't funnel any output to PowerShell's output pipes without dumping the exception to a message queue and then reading it when the execution pipe is running.

I thought about sending it to Trace but in the end because this is a fall back I don't care too much if it doesn't load, I more care to get the assembly not found exception to bubble up and then look at fusionlog to see why it wasn't found in the first place.